src/st_paywall/google_auth.py

```diff
@@ -91,11 +91,11 @@ def get_access_token_from_query_params(
     return token
 
 
-def show_login_button():
+def show_login_button(text: Optional[str] = "Login with Google", color="#FD504D", sidebar: bool = True):
```

Comment:
so looking at this, I think we have to also pass these params to `add_auth()` otherwise users won't really be able to set it

Reply:
let me push a commit

Reply:
Ok I fixed this, before it added the params to show_login_button() but that wouldn't actually do anything because add_auth() is what the developer actually calls, so we want to toss any params in there instead. Merging this now